keras/layers/convolutional.py

```diff
@@ -220,6 +215,8 @@ class Convolution2D(Layer):
         border_mode: 'valid' or 'same'.
         subsample: tuple of length 2. Factor by which to subsample output.
             Also called strides elsewhere.
+        filter_dilation: tuple of length 2. Factor for kernel dilation.
+            Used by dilated convolution(a.k.a. convolution with holes or atrous convolution).
```

Comment:
Sure, thanks for the suggestion, do you think this one https://arxiv.org/abs/1412.7062 is ok?